server/vulnerabilities/nvd/tools/providers/nvd/cve.go

```diff
@@ -229,6 +231,7 @@ func (cf cveFile) baseURL(src SourceConfig) (string, error) {
 }
 
 func (cf cveFile) Sync(ctx context.Context, src SourceConfig, localdir string) error {
+	logger := level.NewFilter(log.NewJSONLogger(os.Stdout), level.AllowInfo())
```

Comment:
creating a new logger here takes logging control out of customer's hands, we should pass the Fleet logger down to this function